database.php

```diff
@@ -85,7 +114,7 @@ function __construct($config) {
 			admin_token TEXT,
 			request_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 			)");
-        $this->db->exec("CREATE TABLE IF NOT EXISTS logins (
+          $this->db->exec("CREATE TABLE IF NOT EXISTS logins (
```

Comment:
This looks like changed indenting. I try to always indent by 4 spaces. I just noticed that I am inconsistent with that as well but I would ask to at least restore the former indentation to reduce diff size

Reply:
The "$this->db->exec" are a now substructure of the mysql-check (or the else-branch of this check). So, if I understand it correctly, they have to be indenting, but 4 spaces. Is this correct?